lib/spack/spack/environment/environment.py

```diff
@@ -2165,14 +2050,15 @@ def is_latest_format(manifest):
 
 
 @contextlib.contextmanager
-def deactivate_environment():
+def environment_deactivated():
```

Comment:
Minor, but I don't like the -ed name here. What about:
```suggestion
def no_active_environment():
```
if the previous name seemed too "imperative"?

Reply:
fixed